bin/webpack.config.js

```diff
@@ -110,7 +91,7 @@ module.exports = (env = {}) => {
                 highlight(code, lang) {
                   const prismLang =
                     Prism.languages[lang] || Prism.languages.clike;
-                  return Prism.highlight(code, prismLang);
+                  return Prism.highlight(code, prismLang, undefined);
```

Comment:
If we add this argument, let's set it with right value (https://github.com/PrismJS/prism/blob/bfd7fded29755510571e3abf0846f2a9edf44ef6/prism.js#L614).
```suggestion
                  return Prism.highlight(code, prismLang, lang);
```